lib/commands/emulate.ts

```diff
@@ -1,68 +0,0 @@
-export class EmulateCommandBase {
-	constructor(private $options: IOptions,
-		private $projectData: IProjectData,
-		private $logger: ILogger,
-		private $platformService: IPlatformService) {
-		this.$projectData.initializeProjectData();
-	}
-
-	public async executeCore(args: string[]): Promise<void> {
-		this.$logger.warn(`Emulate command is deprecated and will soon be removed. Please use "tns run <platform>" instead. All options available for "tns emulate" are present in "tns run" command. To run on all available emulators, use "tns run <platform> --emulator".`);
-		this.$options.emulator = true;
-		const appFilesUpdaterOptions: IAppFilesUpdaterOptions = { bundle: this.$options.bundle, release: this.$options.release };
-		const emulateOptions: IEmulatePlatformOptions = {
-			avd: this.$options.avd,
-			clean: this.$options.clean,
-			device: this.$options.device,
-			release: this.$options.release,
-			emulator: this.$options.emulator,
-			projectDir: this.$options.path,
-			justlaunch: this.$options.justlaunch,
-			availableDevices: this.$options.availableDevices,
-			platformTemplate: this.$options.platformTemplate,
-			provision: this.$options.provision,
-			teamId: this.$options.teamId,
-			keyStoreAlias: this.$options.keyStoreAlias,
-			keyStoreAliasPassword: this.$options.keyStoreAliasPassword,
-			keyStorePassword: this.$options.keyStorePassword,
-			keyStorePath: this.$options.keyStorePath
-		};
-		return this.$platformService.emulatePlatform(args[0], appFilesUpdaterOptions, emulateOptions, this.$projectData, this.$options);
```

Comment:
the `emulatePlatform` method of `platformService` is not used by anyone else, shouldn't we delete it as well?
Also I believe we no longer need `IEmulatePlatformOptions` interface